instance/models/mixins/openedx_database.py

```diff
@@ -235,6 +235,10 @@ def _get_mysql_settings(self):
 
             # ecommerce
             "ECOMMERCE_DEFAULT_DB_NAME": self._get_mysql_database_name("ecommerce"),
+            "ECOMMERCE_DATABASE_NAME": self._get_mysql_database_name("ecommerce"),
+            "ECOMMERCE_DATABASE_USER": self._get_mysql_user_name("ecommerce"),
+            "ECOMMERCE_DATABASE_PASSWORD": self._get_mysql_pass_from_dbname("ecommerce"),
+            "ECOMMERCE_DATABASE_HOST": self.mysql_server.hostname,
             "ECOMMERCE_DATABASES": {
```

Comment:
Similar comment required here..  We keep this stanza for backwards compatibility with Ginkgo.

So to ease confusion, could you also update the `ECOMMERCE_DATABASES["NAME"]`, `["USER"]`, `["PASSWORD"]`, and `["HOST"]` variables below to use the new `ECOMMERCE_DATABASE_NAME`, `_USER`, ... variables you added above?

That way, we can still override individual `ECOMMERCE_DATABASE_NAME` variables in our Ocim instance configuration, and things will continue to work as expected, with both Ginkgo and post-ginkgo releases.

Reply:
Raises a sticky question for Ocim, and its future maintainability, testing, and support when it starts being used by organisations outside of OpenCraft: what versions of the {{edx-platform}} and {{edx:configuration}} repos will we support deployment for?  And how will we ensure that these versions remain compatible with future changes?  CC @antoviaque @smarnach

Reply:
I've also updated the old-style settings to use the new style settings allowing them to be overridden together.

Reply:
@pomegranited In the short term, I suggest we simply keep doing what we did so far:  Keep backwards compatible with the last release, and remove the old stuff when it's not needed anymore.  In the long term, we plan to make Ocim a lot more flexible.  We want to support instance types other than Open edX, and targets other than OpenStack.  At that point, I don't think we will keep the specifics of individual instance types in the core code base – they will be stored externally, e.g. in a different repository or in the database.  We will then be able to have multiple templates for different releases of Open edX, and whenever we want to support a new release, we will add a new template.

This all hasn't been planned or discussed in detail, but that's my idea of where we are going in the long term.